src/App.js

```diff
@@ -8,10 +8,10 @@ import './App.css';
 class App extends Component {
```

Comment:
I can't comment in the first lines. Try to organize your imports depending on the type, your own components, your CSS, your libraries, and then, the react stuff